grpc-protocol/src/main/java/com/linecorp/armeria/client/grpc/protocol/UnaryGrpcClient.java

```diff
@@ -90,9 +113,8 @@ public UnaryGrpcClient(WebClient webClient) {
      */
     public CompletableFuture<byte[]> execute(String uri, byte[] payload) {
         final HttpRequest request = HttpRequest.of(
-                RequestHeaders.of(HttpMethod.POST, uri,
-                                  HttpHeaderNames.CONTENT_TYPE, "application/grpc+proto",
-                                  HttpHeaderNames.TE, HttpHeaderValues.TRAILERS),
+                RequestHeaders.builder(HttpMethod.POST, uri).contentType(serializationFormat.mediaType())
+                              .addObject(HttpHeaderNames.TE, HttpHeaderValues.TRAILERS).build(),
```

Comment:
Could use `add()` to avoid unnecessary object conversion attempt:
```suggestion
                              .add(HttpHeaderNames.TE, HttpHeaderValues.TRAILERS).build(),
```